### ISSUE TITLE
JSON de-serialization fails with single object

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule#379]
**Description of the issue**

When using the JSON format, if a single object is passed on the pipeline, de-serialization fails.

For example:

```powershell
"{ `"key`": `"value`" }" | Invoke-PSRule -Format json;
```

This does not occur when using YAML or when the JSON object is nested in an array.

**To Reproduce**

Steps to reproduce the issue:

```powershell
 kubectl get services -o json | Out-String | Invoke-PSRule -Format json -ObjectPath items -Module PSRule.Rules.Kubernetes;
```

**Expected behaviour**

The object should be detected as a single instance and de-serialized regardless.

**Error output**

```text
Read JSON failed.
At C:\Users\<removed>\Documents\PowerShell\Modules\PSRule\0.13.0\PSRule.psm1:183 char:17
+                 $pipeline.Process($InputObject);
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (:) [], PipelineSerializationException
+ FullyQualifiedErrorId : PSRule.Pipeline.PipelineSerializationException
```

**Module in use and version:**

- Module: PSRule
- Version: **0.13.0-B1912043**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      6.2.3
PSEdition                      Core
GitCommitId                    6.2.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
